themes/theme-ios11/pages/More/components/UserMenu/components/LoggedIn/index.jsx

```diff
@@ -14,40 +15,38 @@ import Item from '../../../Item';
 const LoggedIn = ({ logout }) => {
```

Comment:
maybe you feel like updating the JSDoc correctly..

Reply:
done